test-clients/src/main/java/com/hedera/services/bdd/spec/queries/contract/HapiContractCallLocal.java

```diff
@@ -165,6 +165,16 @@ private Query getContractCallLocal(HapiApiSpec spec, Transaction payment, boolea
 		return Query.newBuilder().setContractCallLocal(query).build();
 	}
 
+//	private void validateAddressSize(int size) {
+//		int retVal = 0;
+//		ByteString functionalResults = response.getContractCallLocal().getFunctionResult().getContractCallResult();
+//		byte[] result = functionalResults.toByteArray();
+//		if (result != null && result.length > 0) {
+//			retVal = decodeGetCodeSizeResult(result);
+//		}
+//
+//	}
+
```

Comment:
Is the commented out method still needed?

Reply:
removing it.